### PR TITLE
Change firewall_apply behavior

### PR DESCRIPTION
**Pull request by acalm**
As title suggests, the function will not do any kind of formatting and just pass address groups, networks and rulesets on to Ansible

### CODE REVIEW

File: socrates_api/serializers.py [hunk 1]
```diff
@@ -701,7 +701,7 @@ def update(self, instance, data):
                     pass
             else:
                 continue
-            firewall_apply.apply_async((asset, network))
+            firewall_apply.apply_async((asset))
```
Comment (dhozac) [inline socrates_api/serializers.py]: This is missing a trailing comma to make it a tuple.

File: socrates_api/serializers.py [hunk 2]
```diff
@@ -766,7 +766,7 @@ def update(self, instance, validated_data):
                 except:
                     pass
                 else:
-                    firewall_apply.apply_async((asset, network))
+                    firewall_apply.apply_async((asset))
```
Comment (dhozac) [inline socrates_api/serializers.py]: Same here.

**Review by dhozac — CHANGES_REQUESTED** [review]
Looks good! I think this means we can probably drop some of the resolve code from the serializers as well, if you want to get an even higher score!

**Comment by acalm** [comment]
> Looks good! I think this means we can probably drop some of the resolve code from the serializers as well, if you want to get an even higher score!

Well, I do like higher scores!

**Comment by dhozac** [comment]
This will need to update testdata/playbooks/firewall-apply.yml as well to look for the new variables.

**Pull request merged**